src/engine/strat_engine/backstore/metadata/mda.rs

```diff
@@ -574,9 +602,10 @@ mod tests {
     #[test]
     fn test_from_buf_crc_error() {
         let data = [0u8; 3];
+
         let header = MDAHeader {
             last_updated: Utc::now(),
-            used: Bytes(data.len() as u64),
+            used: MetaDataSize::new(Bytes(data.len() as u64)),
```

Comment:
The Jenkins test failed with this error:
```
error[E0308]: mismatched types
   --> src/engine/strat_engine/backstore/metadata/mda.rs:608:19
    |
608 |             used: MetaDataSize::new(Bytes(data.len() as u64)),
    |                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |                   |
    |                   expected enum `std::option::Option`, found struct `engine::strat_engine::backstore::metadata::mda::MetaDataSize`
    |                   help: try using a variant of the expected type: `engine::strat_engine::serde_structs::_IMPL_SERIALIZE_FOR_PoolSave::_serde::export::Some(<MetaDataSize>::new(Bytes(data.len() as u64)))`
    |
    = note: expected type `std::option::Option<engine::strat_engine::backstore::metadata::mda::MetaDataSize>`
               found type `engine::strat_engine::backstore::metadata::mda::MetaDataSize`

error: aborting due to previous error
```

Did you mean to say this?
```
            used: Some(MetaDataSize::new(Bytes(data.len() as u64))),
```

Reply:
Weird that I didn't get the error when running locally `make build`... I'll update, thanks for the logs!

Reply:
Ah it was in a test, that's why!

Reply:
> Ah it was in a test, that's why!

```
> make test
```

will compile all the tests and run the ones that are purely computational (don't build devices or actually have an effect on your machine). I always make sure not to be root when I do this on my own machine, just in case there's an error in how the tests are set up, but that has never been the case so far.